.github/workflows/lint-test.yml

```diff
@@ -66,22 +66,14 @@ jobs:
       - name: Migrate
         run: go run main.go migrate_up
         env:
-          POSTGRES_USER: root
-          POSTGRES_PASSWORD: mysecret
-          DB_NAME: cdn
-          DB_HOST: localhost
-          DB_PORT: 5432
+          DB_URI: postgres://root:mysecret@localhost:5432/cdn?sslmode=disable
```

Comment:
Can we please use constants for this over all repositories?

postgres://twt:twt@postgres:5432/twt